docs/compliance/index.md

```diff
@@ -0,0 +1,386 @@
+- This foundational architecture provides a baseline to help customers use Microsoft Azure in a PCI DSS-compliant manner.
+- Achieving PCI DSS-compliance requires that an accredited Qualified Security Assessor (QSA) certify a production customer solution.
+- Customers are responsible for conducting appropriate security and compliance reviews of any solution built using this foundational architecture, as requirements may vary based on the specifics of each customer’s implementation and geography.  
+
+## Architectural Diagram
+
+![](images/pci-architectural-diagram.png)
+
+## Solution Components
+
+The foundational architecuture is comprised of the following components:
+
+- **Architectural diagram**. The diagram shows the reference architecture used for the Contoso Webstore solution.
+- **Deployment templates**. In this deployment, [Azure Resource Manager templates](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#template-deployment) are used to automatically deploy the components of the architecture into Microsoft Azure by specifying configuration parameters during setup.
+- **Automated deployment scripts**. These scripts help deploy the end-to-end solution. The scripts consist of:
+    - A module installation and [global administrator](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-assign-admin-roles-azure-portal) setup script is used to install and verify that required PowerShell modules and global administrator roles are configured correctly.
+    - An installation PowerShell script is used to deploy the end-to-end solution, including [security components](https://github.com/Microsoft/azure-sql-security-sample) built by the Azure SQL Database team. 
+
+## Deploying the Solution
+
+The deployment of the automated foundational architecture requires several steps executed via Microsoft PowerShell v5. To connect to the website, you must provide a custom domain name (such as contoso.com). This is specified using the `-customHostName` switch in step 2. For more information, see [Buy a custom domain name for Azure Web Apps](https://docs.microsoft.com/en-us/azure/app-service-web/custom-dns-web-site-buydomains-web-app).
+A custom domain name is not required to successfully deploy and run the solution, but you will be unable to connect to the website for demonstration purposes.
+
+It is highly recommended that a clean installation of PowerShell be used to deploy the solution. Alternatively, verify that you are using the latest modules required for proper execution of the installation scripts. In this example, we log into a Windows 10 virtual machine and execute the following commands (note that this enables the custom domain command):
+
+1. Install required modules and set up the administrator roles correctly.
+```powershell
+ .\0-Setup-AdministrativeAccountAndPermission.ps1 
+    -azureADDomainName contosowebstore.onmicrosoft.com
+    -tenantId XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
```

Comment:
Maybe add a note that describes how to find your tenant ID via the portal (AAD > Properties > Directory ID)